File: Clases 2018/Clases-C#/Usuarios+SeleccionActBugs/App_BugTracker resuelto/App_BugTracker/BusinessLayer/Entites/Bug.vb
Public Class Bug
    Public Property id_bug As Integer
    Public Property titulo As String
    Public Property descripcion As String
    Public Property id_producto As Integer
    'Nombre producto
    Public Property n_producto As String
    Public Property id_prioridad As Integer
    'Nombre prioridad
    Public Property n_prioridad As String
    Public Property id_criticidad As Integer
    'Nombre criticidad
    Public Property n_criticidad As String
    'Nombre del último estado o estado actual del bug
    Public Property estado As String
    'Fecha de alta o fecha de creación. 
    Public Property fecha_alta As Date
    Public Property asignado_a As String

    Public Property historial As List(Of HistorialBug)

    Public Sub New()
        historial = New List(Of HistorialBug)
    End Sub

    Public Sub addHistorial(ByVal oHistorialBug As HistorialBug)
        historial.Add(oHistorialBug)
    End Sub

End Class

File: Clases 2018/Clases-C#/ClasePractica10_2018 (update tabla Bugs)/C#_TPS_BugTracker/TPS_BugTracker/BusinessLayer/Entities/Bug.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TPS_BugTracker.BusinessLayer.Entities
{
    public class Bug
    {

        public int id_bug { get; set; }
        public string titulo { get; set; }
        public string descripcion { get; set; }
        public int id_producto { get; set; }
        // Nombre producto
        public string n_producto { get; set; }
        public int id_prioridad { get; set; }
        // Nombre prioridad
        public string n_prioridad { get; set; }
        public int id_criticidad { get; set; }
        // Nombre criticidad
        public string n_criticidad { get; set; }
        // Nombre del último estado o estado actual del bug
        public string estado { get; set; }
        // Fecha de alta o fecha de creación. 
       
        public DateTime fecha_alta { get; set; }
        public string asignado_a { get; set; }

        public List<HistorialBug> historial { get; set; }

        public Bug()
        {
            historial = new List<HistorialBug>();
        }

        public void addHistorial(HistorialBug oHistorialBug)
        {
            historial.Add(oHistorialBug);
        }
    }
}
